server/src/main/java/com/linecorp/centraldogma/server/command/AbstractCommandExecutor.java

```diff
@@ -126,7 +126,7 @@ public final <T> CompletableFuture<T> execute(Command<T> command) {
         if (!isStarted()) {
             throw new ReadOnlyException("running in read-only mode. command: " + command);
         }
-        if (!writable && !(command instanceof AdministrativeCommand)) {
+        if (!writable && !(command instanceof SystemAdministrativeCommand)) {
```

Comment:
Double checked that the superclass `[System]AdministrativeCommand` isn't exposed in the JSON in any way